amplify/backend/api/colonycdapp/schema/schema.graphql

```diff
@@ -973,22 +973,30 @@ type Mutation {
   updateContributorsWithReputation(
     input: UpdateContributorsWithReputationInput!
   ): Boolean @function(name: "updateContributorsWithReputation-${env}")
-
   """
   Validates the user invite code and adds the user as a colony contributor
   """
   validateUserInvite(input: ValidateUserInviteInput!): Boolean
     @function(name: "validateUserInvite-${env}")
   """
+  Initialize user (for now only cancels pending transactions)
```

Comment:
I gave this a bit of a more general name to be able to add more to this lambda if we so wish